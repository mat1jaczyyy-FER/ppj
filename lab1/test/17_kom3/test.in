a = x+y // x + y
b = 12 // &^#$ bilo sto
// prazan redak
// broji retke ispravno
x + 3
