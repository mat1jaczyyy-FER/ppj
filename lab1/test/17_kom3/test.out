IDN 1 a
OP_PRIDRUZI 1 =
IDN 1 x
OP_PLUS 1 +
IDN 1 y
IDN 2 b
OP_PRIDRUZI 2 =
BROJ 2 12
IDN 5 x
OP_PLUS 5 +
BROJ 5 3
